src/ds/features/auto-exposure-roi-feature.h

```diff
@@ -0,0 +1,31 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <src/feature-interface.h>
+
+#include <string>
```

Comment:
I don't see any usage here...
You do need `<memory>`, though.

Reply:
Fixed